releasenotes/notes/rollback_if-039fc1c97caeed37.yaml

```diff
@@ -0,0 +1,22 @@
+---
+features:
+  - |
+    A new pass manager controller `rollback_if` allows to "undo" passes under certain conditions.
+    If the condition is true the passes appended in that set are rolled back. Take for example a
+    pass the processes of choosing the best swapper for a particular situation.
+    The following pass manager applies ``Swapper1`` and calculates the depth of the swapper, saving
+    The result in ``property_set['depth_swapper1']``:
+
+    ```python
+    passmanager = PassManager()
+    passmanager.append([Swapper1(), depth('depth_swapper1')])
```

Comment:
pretty sure this is not how the `depth` pass works. You can't just pass it an argument like that. Can you just write a real example, and actually use `StochasticSwap` vs. `LookaheadSwap`? The release note should make it easy for people to try it out and see how it works.